mne_bids/utils.py

```diff
@@ -37,18 +37,18 @@ def print_dir_tree(dir):
             print('|%s %s' % (len(path) * '---', file))
 
 
-def _mkdir_p(path, overwrite=False, verbose=False):
+def _mkdir_p(path, write_mode='append', verbose=False):
```

Comment:
`'append'` is used in this private function? If not, why is this the default ...